.github/workflows/updater.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+#=================================================
+# PACKAGE UPDATING HELPER
+#=================================================
+
+# This script is meant to be run by GitHub Actions
+# The YunoHost-Apps organisation offers a template Action to run this script periodically
+# Since each app is different, maintainers can adapt its contents so as to perform
+# automatic actions when a new upstream release is detected.
+
+#=================================================
+# FETCHING LATEST RELEASE AND ITS ASSETS
+#=================================================
+
+# Fetching information
+current_version=$(cat manifest.json | jq -j '.version|split("~")[0]')
+# Some jq magic is needed, because the latest upstream release is not always the latest version (e.g. security patches for older versions)
+version=$(curl --silent "https://git.pleroma.social/api/v4/projects/2/releases" | jq -r '.[] | select( .upcoming_release != true ) | .tag_name' | sort -V | tail -1)
+assets=($(curl --silent "https://git.pleroma.social/api/v4/projects/2/releases" | jq -r '[ .[] | select(.tag_name=="'$version'").assets.sources[].url ] | join(" ") | @sh' | tr -d "'"))
+release=$(curl --silent "https://git.pleroma.social/api/v4/projects/2/releases" | jq -r '[ .[] | select(.tag_name=="'$version'")._links.self ] | join(" ") ' | tr -d "'")
+description=$(curl --silent "https://git.pleroma.social/api/v4/projects/2/releases" | jq -r '[ .[] | select(.tag_name=="'$version'").description ] | join(" ") ' | tr -d "'")
+commit_id=$(curl --silent "https://git.pleroma.social/api/v4/projects/2/releases" | jq -r '[ .[] | select(.tag_name=="'$version'").commit.id ] | join(" ") ' | tr -d "'")
+pipeline_id=$(curl --silent "https://git.pleroma.social/api/v4/projects/2/pipelines" | jq -r '[ .[] | select((.ref=="stable") and (.sha=="'$commit_id'")).id|tostring ] | join(" ") ' | tr -d "'")
```

Comment:
```suggestion
pipeline_id=$(curl --silent "https://git.pleroma.social/api/v4/projects/2/pipelines?sha=$commit_id&ref=stable" | jq -r '[ .[] | .id|tostring ] | join(" ") ' | tr -d "'")
```

Since this endpoint of the API supports it, use query strings to optimize this API endpoint call to only get the pipeline related to whatever commit and the stable branch.